### ISSUE TITLE
Rename library packages to bitcoincashj

### ISSUE THREAD

**Issue by Tolsi** [bitcoinj-cash/bitcoinj#25]
Now it's impossible to use this module with bitcoinj in one project due the package name conflict, this is very inconvenient.

**Comment by HashEngineering** [comment]
How could we meet your needs of renaming the packages while still meeting the needs of the other developers who need the package name to stay the same?

**Comment by Tolsi** [comment]
This is the different code base for bitcoin and bitcoin cash, why you need it to be in the same package? I think a couple of people using this bitcoinj will do just that.

**Comment by HashEngineering** [comment]
For these purposes, it is easier to leave the package names the same:
- pulling updates from bitcoinj for version 0.15 
- Forking other bitcoin only software to BCH that uses bitcoinj

We are open to solutions to meeting both needs.  It has been discussed in our mailing list.

**Comment by Tolsi** [comment]
That's really strange lazy solution.

- This is slightly hampering you to pull changes from the original bitcoinj, because there never changes the package. If there are new classes, then the package is easy to rename by the text replacement. Do you often do this?
- Forking other bitcoin only software to BCH must be explicit, and not just a change in the dependencies.

**Comment by kodmanyagha** [comment]
Yes the package names must be change. This is bitcoincash library and must be different from bitcoinj. It's enough change prefix. For example change org.bitcoinj to org.bitcoincashj is enough.

**Comment by HashEngineering** [comment]
This topic has been discussed in the mailing list:
https://groups.google.com/forum/#!forum/bitcoinj-cash

Our decision last year was not to change the package name.  None of our projects that use this library required that change at that time.  As one would notice from the level of activity on this repo and the mailing list, the developers involved have limited time to work on this project.  Things will be moving slowly.

Our idea for renaming the packages would be to cash.bitcoinj.* following the same name as the groupId in the maven repo.  Ideally we would support a library for org.bitcoinj (for the lazy and strange developers) and cash.bitcoinj for multicoin app developers -- one codebase would be maintained primarily on the repo, while the other is generated using a script that changes all the names.  Again, these are all ideas on paper with few resources to accomplish things as quickly as some may like.  We welcome pull requests and testers and opinions in the mailing list.
